http/src/main/resources/reference.conf

```diff
@@ -241,11 +241,11 @@ gke {
     ]
   }
   galaxyApp {
-    releaseNameSuffix = "galaxy-rls"
+    releaseNameSuffix = "gxy-rls"
```

Comment:
See comment in `KubernetesServiceInterp` for more context. Theoretically release names _should_ only need to be unique within a namespace, but something in the Galaxy chart requires them to be unique within a cluster.

Reply:
Sure